helpers/commit_helpers.rb

```diff
@@ -0,0 +1,16 @@
+module CommitHelpers
+  def commit_url(current_page)
+    commit_sha = `git log -1 --format=oneline #{source_file(current_page)} | cut -d ' ' -f 1`.strip
```

Comment:
Nice one working these out - I didn't appreciate how complicated it would be to just print two things about each file! And due to the limitations of middleman for these files, I agree that a helper seems the best place to put the code.

I warmed to this even more when I realised we can en-neaten this a bit with the git gem e.g.

```
Git.open('.').log.path("README.md").first.sha
```

Reply:
That does look neater. We don't have [ruby-git](https://github.com/ruby-git/ruby-git) installed yet - is it worth introducing a dependency just to simplify one line of code? I would lean towards leaving this as-is, but don't feel strongly either way.

Reply:
Ah, good point. I think it's worth the effort in this case:

- It's a small dependency, so shouldn't impact day-to-day development
- It's less coupled to command line options and other commands (`cut`).
- It's a bit of future-proofing against doing any more `git` commands.